src/pages/groupComparison/GroupComparisonPage.tsx

```diff
@@ -196,42 +197,42 @@ export default class GroupComparisonPage extends React.Component<IGroupCompariso
         } 
     });
 
+    @computed get overlapStrategySelector() {
+        if (this.store._selectionInfo.isComplete) {
+            if (this.store._selectionInfo.result.overlappingSamples.length === 0 &&
+            this.store._selectionInfo.result.overlappingPatients.length === 0) {
+                return null;
+            } else {
+                // (${caseCountsInParens(this.store._selectionInfo.result.overlappingSamples, this.store._selectionInfo.result.overlappingPatients)})
+                return (
+                    <div style={{width:333, zIndex:20}}>
+                        <ReactSelect
```

Comment:
you might as well use new ReactSelect2 to avoid having to update later.  probably not a big deal to change it.

Reply:
kk